### ISSUE TITLE
consider adding to the community themes?

### ISSUE THREAD

**Issue by psifertex** [dracula/binary-ninja#2]
I realize there's already an existing dracula theme, but would you consider adding this as a second dracula theme to the repo?

https://github.com/Vector35/community-themes

**Comment by CX330Blake** [comment]
Yes of course! It's my honor, thx for asking. I'll keep the issue open til you add it on, feel free to ask anything!

**Comment by psifertex** [comment]
Sorry, didn't close this out yet. Do you have a preferred name for the plugin on the main repo? "Dracula's Revenge" maybe?

**Comment by psifertex** [comment]
Or maybe "Dracula Pro"?

**Comment by CX330Blake** [comment]
> Sorry, didn't close this out yet. Do you have a preferred name for the plugin on the main repo? "Dracula's Revenge" maybe?

I think I will go with "Dracula's Revenge", sounds cool!

**Comment by psifertex** [comment]
Done! Added, sorry for taking so long.

https://github.com/Vector35/community-themes/commit/9c43434e6b48f86da9e19527ad854ccefc8a04e1
